### PR TITLE
Fixed h3 closing in index.ejs to pass validation

### PR DESCRIPTION
**Pull request by Qazare**
I followed your instructions and modified the index.ejs file this time, and checked W3C validation on my own copy's page. It seems all good now !

### CODE REVIEW

File: src/index.ejs [hunk 1]
```diff
@@ -12,7 +12,7 @@
 <body>
   <div id="map-host"></div>
   <div id="toast">
-    <h3 id="toast-message" />
+    <h3 id="toast-message"></h3>
```
Comment (Vazerthon) [inline src/index.ejs:15]: 👍

**Review by Vazerthon — COMMENTED** [review]
That's looking good. Can you please remove the DS_Store files and package-lock.json?

**Comment by Qazare** [comment]
Done ! :beers:

**Review by Vazerthon — APPROVED** [review]
Perfect. Thanks for your input :-)

**Pull request merged**